src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -213,6 +218,15 @@ internal partial class CodeFixService : ForegroundThreadAffinitizedObject, ICode
             return result.ToImmutableAndFree();
         }
 
+        public async Task<CodeFixCollection> GetFixesAsync(Document document, TextSpan range, string diagnosticId, CancellationToken cancellationToken)
+        {
+            var fixesCollectionArray = await GetFixesAsync(document, range, includeSuppressionFixes: false, diagnosticId, cancellationToken).ConfigureAwait(false);
+
+            // TODO: Just get the first fix for now until we have a way to config user's preferred fix
+            // https://github.com/dotnet/roslyn/issues/27066
+            return fixesCollectionArray.FirstOrDefault();
```

Comment:
what is the impact of this problem?

Reply:
it always takes the first fix, i guess it is better than not taking anything.